Softeq.XToolkit.PushNotifications.iOS/IosNotificationCategoriesProvider.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Foundation;
+using UIKit;
+using UserNotifications;
+
+namespace Softeq.XToolkit.PushNotifications.iOS
+{
+    public class IosNotificationCategoriesProvider : INotificationCategoriesProvider
+    {
+        public IList<UNNotificationCategory> NotificationCategories { get; } = new List<UNNotificationCategory>();
+
+        protected readonly Dictionary<string, IList<UNNotificationAction>> _actionsForCategories = new Dictionary<string, IList<UNNotificationAction>>();
```

Comment:
Fields should be defined before properties. Also, does it really have to be protected? Seems like we already define all methods to manipulate this dictionary, so it shouldn't be modified directly

Reply:
Fixed